Makefile

```diff
@@ -48,3 +48,13 @@ build-7-3-24:
 
 build-7-4-6:
 	docker build -t sparkfabrik/php7.4.6 7.4.6-fpm-alpine3.10
+
+build-7-4-16-rootless:
+	docker build -t sparkfabrik/php7.4.16-rootless 7.4.16-fpm-alpine3.13-rootless
+	@if [ $$(docker run --rm sparkfabrik/php7.4.16-rootless ash -c "php-fpm -i | grep 'PHP_VERSION => '" | awk '{print $$3}') = "7.4.16" ]; then \
+		echo "The image was built successfully"; \
+		exit 0; \
+	else \
+		echo "Error in the build process"; \
+		exit 1; \
+	fi
```

Comment:
i think this is the right approach, what we can do is to move the tests in a separate script so that can be reused also for the other builds.
What we should do as well is to test all the entrypoint parameters, just to be sure that everything is working as expected.